disperser/dataapi/metrics.go

```diff
@@ -26,9 +26,13 @@ type Metrics struct {
 
 	NumRequests    *prometheus.CounterVec
 	Latency        *prometheus.SummaryVec
-	Semvers        *prometheus.GaugeVec
 	OperatorsStake *prometheus.GaugeVec
 
+	Semvers                *prometheus.GaugeVec
+	SemversStakePctQuorum0 *prometheus.GaugeVec
+	SemversStakePctQuorum1 *prometheus.GaugeVec
+	SemversStakePctQuorum2 *prometheus.GaugeVec
```

Comment:
in the future, when there are more custom quorums, we need to abstract it a bit more